site/docs/api/table-options.md

```diff
@@ -496,6 +496,20 @@ The table options are defined in `jQuery.fn.bootstrapTable.defaults`.
 
 - **Example:** [Only Info Pagination](https://examples.bootstrap-table.com/#options/only-info-pagination.html)
 
+## showExtendedPagination
+
+- **Attribute:** `data-show-extended-pagination`
+
+- **Type:** `Boolean`
+
+- **Detail:**
+
+  Set `true` to show a extended version of pagination (including the count of all rows with out filters).
+
+- **Default:** `false`
+
+- **Example:** [Only Info Pagination](https://examples.bootstrap-table.com/#options/show-extended-pagination.html)
```

Comment:
Example error.

We need to add `totalNotFilteredField` and `totalNotFiltered` option to docs.

Reply:
Why totalNotFiltered ?
Its just a variable which contains the amount of , it makes no sense to change that (in my opinion).

totalNotFilteredField done.